.github/workflows/scripts/golang_check_version.sh

```diff
@@ -1,71 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
This script is still relevant, maybe it can go into another workflow if the rest of the workflow file was just for /src/go.

Reply:
How about a workflow like this?
```
name: AGW Check Go Version
on:
  push:
    branches:
      - master
      - 'v1.*'
  pull_request:
    branches:
      - master
      - 'v1.*'
    types:
      - opened
      - reopened
      - synchronize

concurrency:
  group: ${{ github.workflow }}-${{ github.event.pull_request.number || github.sha }}
  cancel-in-progress: true

jobs:
  check_go_version:
    runs-on: ubuntu-20.04
    env:
      MAGMA_ROOT: "${{ github.workspace }}"
    steps:
      - uses: actions/checkout@93ea575cb5d8a053eaa0ac8fa3b40d7e05a33cc8 # pin@v3.1.0
      - name: Run golang_check_version.sh script
        run: ./.github/workflows/scripts/golang_check_version.sh

```

Reply:
Added the file back. Will handle the workflow in a separate PR.

Reply:
Link to PR https://github.com/magma/magma/pull/14592